tests/unit_tests/test_wave_01.py

```diff
@@ -49,7 +49,5 @@ def test_removing_not_found_is_false():
 
     result = vendor.remove(item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result is False
+    assert len(vendor.inventory) == 3
```

Comment:
👍 Checking that the length is unchanged is probably fine, but in some tests, we might want to check that nothing unexpected happened to the particular items in the list. Yes, there are three, but which three?

From one perspective, this might be too pessimistic. After all, why would trying to remove a non-existent value from a collection result in any of the items it doesn't match being changed. From another perspective, this might be too optimistic. Maybe we should check that the original three values are unchanged (not just that there are still three values at all). Tests can check a variety of conditions, and while not every test needs to check every condition, it's not a bad idea to have an occasional test that's a bit more suspicious of the results.